resources/etc/systemd/system/docker-metadata.service.d/docker-start-script.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+TARGET_FILE=/etc/systemd/system/docker-metadata.service.d/docker-metadata-environment
+
+function get_enabled(){
+  etcdctl get config/_global/proxy/enabled || echo "false"
+}
+
+if [ "$(get_enabled)" == "true" ]
+then
+  echo "collecting https proxy config steps..."
+  HOST="$(etcdctl get config/_global/proxy/server)" || (echo "ERROR: Could not read from etcd: proxy host" && exit 0)
+  PORT="$(etcdctl get config/_global/proxy/port)" || (echo "ERROR: Could not read from etcd: proxy port" && exit 0)
+  USERNAME="$(etcdctl get config/_global/proxy/username)" || echo "WARNING: Could not read from etcd: proxy host"
+  PASSWORD="$(etcdctl get config/_global/proxy/password)" || echo "WARNING: Could not read from etcd: proxy host"
+  AUTH=""
+  if [ "${USERNAME}" != "" ] && [ "${USERNAME}" != "" ]
+  then
+    AUTH="${USERNAME}:${PASSWORD}@"
+  else
+    echo "No username and password for proxy configured."
+  fi
+
+  HTTP_CONFIG="HTTP_PROXY=http://${HOST}:${PORT}"
```

Comment:
Is there no authentication required for `HTTP_PROXY`?

Reply:
This code has been removed.